src/plugins/wizard/public/application/utils/state_management/metadata_slice.ts

```diff
@@ -38,11 +40,14 @@ export const slice = createSlice({
       const { key, valid } = action.payload;
       state.editorState.validity[key] = valid;
     },
+    setSaveState: (state, action: PayloadAction<{ state: 'initial' | 'clean' | 'dirty' }>) => {
```

Comment:
can we call this `setEditorState`? since the property is quite literally `editorState.state` 😂

Reply:
haha yea that makes sense :)